shina.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Documentations are usually written in Markdown. Switch to using a markdown. See [sample](https://github.com/Stutern/eng-practices/blob/master/README.md)